lms/static/scripts/frontend_apps/components/Button.js

```diff
@@ -26,5 +28,6 @@ Button.propTypes = {
   disabled: propTypes.bool,
   label: propTypes.string,
   onClick: propTypes.func,
+  ref: propTypes.object,
```

Comment:
Effectively this is a pass through ref down to the button, but I could not find another example of doing this in the code which made me question it entirely. Otherwise, I think we would have to account for the whole `ref.current.base` vs `ref.current` between an JSX Element and Element and I'd rather not expose that to `<Dialog>`